x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/job/NodeLoad.java

```diff
@@ -25,7 +25,9 @@ public class NodeLoad {
     private final String nodeId;
     private final boolean useMemory;
     private final String error;
-    private final int numAssignedJobs;
+    private int numAssignedAnomalyDetectorJobs;
+    private int numAssignedDataFrameAnalyticsJobs;
+    private int numAssignedNativeInferenceJobs;
```

Comment:
```suggestion
    private final int numAssignedAnomalyDetectorJobs;
    private final int numAssignedDataFrameAnalyticsJobs;
    private final int numAssignedNativeInferenceJobs;
```

Everything in NodeLoad should be final.

Reply:
🤦